### PR TITLE
Add higher contrast error color

### PR DESCRIPTION
**Pull request by martinwheeler**
# Proposed Changes

This PR aims to improve the readability of the error messages that appear in the status bar.

I have changed the text color to [#FFBC00](https://www.colorhexa.com/ffbc00) which is subtle but still has decent contrast with the blue background.

This PR resolves issue https://github.com/ShailenNaidoo/Netlify/issues/6

I tested it by replacing the listening message color as I wasn't able to replicate an error message state.

Screen Shot 2019-12-19 at 4 41 31 pm

FWIW I used this tool to help me find a color: https://paletton.com/#uid=33v0u0k++JguxZLP++V+WtF+9ng

### CODE REVIEW

**Comment by ShailenNaidoo** [comment]
@martinwheeler Thanks man! I will check this super slick PR this evening, appreciate it :smile:

**Comment by ShailenNaidoo** [comment]
@martinwheeler I had a look at the PR!  The colour works nicely on blue and black but not as nice on light backgrounds but we can always look into that in the future :smile:

File: src/netlify_eventobserver.ts [hunk 1]
```diff
@@ -61,14 +63,14 @@ netlifyEvents.on('error', ({ branch, context }) => {
   logOutputMessage(`Failed to deploy vscode.ThemeColor${branch} to ${context}`);
```
Comment (ShailenNaidoo) [inline src/netlify_eventobserver.ts]: @martinwheeler Do you mind removing `vscode.ThemeColor` from this line? I must have pasted it there by accident :sweat_smile:

Comment (martinwheeler) [reply src/netlify_eventobserver.ts]: Hey @ShailenNaidoo this is now done. Sorry it took so long I had a very long break over the holiday period.

**Comment by ShailenNaidoo** [comment]
Thanks Martin, I will merge it in today

**Comment by ShailenNaidoo** [comment]
@martinwheeler Thanks for contribution man! :smile:

**Pull request merged**